### PR TITLE
feat(condo): INFRA-469 get rid of deprecated RegisterBillingReceiptsS…

### PR DESCRIPTION
**Pull request by ekabardinsky**
…ervice flow with consumer provided normalized address

### CODE REVIEW

File: apps/callcenter
Comment (Alllex202) [inline apps/callcenter:1]: Are these changes part of your pr?

File: apps/condo/domains/ticket/components/IncidentForm/BaseIncidentForm.tsx
Comment (Alllex202) [inline apps/condo/domains/ticket/components/IncidentForm/BaseIncidentForm.tsx:1]: And these?

File: apps/callcenter
Comment (ekabardinsky) [reply apps/callcenter:1]: nope, seems like a mistake of rebase from main

File: apps/condo/domains/ticket/components/IncidentForm/BaseIncidentForm.tsx
Comment (ekabardinsky) [reply apps/condo/domains/ticket/components/IncidentForm/BaseIncidentForm.tsx:1]: nope, seems like a mistake of rebase from main

**Review by dkoviazin — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**